src/lib/process/api.ts

```diff
@@ -43,6 +44,11 @@ export const createProcessTransaction = async (
             program,
           }
 
+          if(verbose == false)
+          {
+            delete newProcess.program
```

Comment:
purely FYI: https://dev.to/hardiquedasore/avoid-delete-keyword-in-javascript-to-remove-property-32mf there are a number of opinions but for now if it works it works